doc/rfd/rfd/0009/README.md

```diff
@@ -0,0 +1,176 @@
+chart: this simplifies consumption downstream.
+
+While targets are allowed to take arguments and options on execution besides the
+expected default behaviour defined here, this is discouraged.
+
+Each target is expected to be a folder (E.g: `install/`) containing executable
+files starting with digits (E.g: `10_do-foo.sh`, `20_do-bar.rb`).
+
+Upon target call, all the executable files starting with digits will get
+executed in alphanumeric order.
+It is allowed for the folder to contain other files/folders besides those
+starting with digits. Those would not be automatically executed upon target
+call (e.g: readme files, local include folders, etc).
+  
+Deployment and automation scripts will expose the following targets:
+
+- `chart`: Obtains the latest public release of the chart at hand, and puts it
+  already decompressed in `$pwd/$our-chart/chart/`. `$our-chart` being the name
+  of the chart at hand (e.g: kubecf).
+
+- `configure`: Outcome is a valid `$pwd/$our-chart/values.yaml` file for the
+  chart at hand. This target accepts a list of alphanumerically ordered subsets
+  of `values.yaml`, inside the folder `configure/patches/`, which get merged to
+  output the final values.yaml. This allows for reuse of existing example
+  values, and for downstream tools to inject their own yaml subsets as needed.
+
+- `install`: Takes a `$pwd/$our-chart/values.yaml` and the chart contained in
+  `$pwd/$our-chart/chart/` and installs the Helm Chart at hand. 
+
+- `wait`: Will wait until deployment of the chart is up and running.
+
+- `upgrade`: Takes a `$pwd/values.yaml` and a chart contained in
+  `$pwd/$our-chart/chart/` and upgrades the Helm Chart at hand. The upgrade must
+  work for both same version and different versions.
+
+- `clean`: Removes all possible objects created by any target from the cluster.
+   Deletes `$pwd/$chart-name` at the end.
+
+- `one-offs/*`: Helper targets. Follow the same rules as the rest of targets,
+  except they may not be idempotent. E.g: `klog`, `login`, `smokes`, `cats`.
+  
+Each of these targets:
+
+- Assumes the deployment/testing env is already loaded, with access to kube, cf,
+  helm operations.
+- Returns retcode `0` on success, `!= 0` if otherwise.
+- Consumes and creates files from `pwd`.
+- Is idempotent: running it will give you the same result. E.g: calling
+  `configure` several times with the same outputs gives you the same
+  `values.yaml` as result. Calling `install` with the same `values.yaml` gives
+  you the same deployment (that means that `install` must clean on its own).
+- Is isolated: implementation of `install` does not call on `configure` itself.
+  They are allowed though to call shared implementation code in the `include`
+  folder of their shared root path.
+- When useful, saves resulting artifacts of execution into `$pwd/artifacts`.
+  E.g: `$pwd/artifacts/cats-run.log`
+
+
+Apart from the targets, the following folders may be provided:
```

Comment:
I think these _must_ be provided. They must be used to compose the helm install/upgrade commands.

Reply:
Correct! fixed. Also, I added imagelist.txt.